src/imagery/i.eodag/i.eodag.py

```diff
@@ -98,6 +98,14 @@
 # % guisection: Region
 # %end
 
+# %option
+# % key: pattern
+# % type: string
+# % description: Filter products by title name (using regex)
```

Comment:
```suggestion
# % description: Filter products by title name using a regular expression, e.g. 'S1.*_SDV.*'
```
An example would help in the description, e.g. as suggested above, but you may have a more relevant one...

Reply:
BTW. the term "title name" is a bit confusing, at least to me. Is "file name" a correct interpretation?

Reply:
There are two terms:
Title: Name of the product on a provider, and it can vary for different providers...
UUID: Unified for a given product across different providers.

Title is actually what we pass to the `id` option, when we are trying to download products by their id.
So I will just change it to: 

> Filter products by id using a regular expression, e.g. 'LC09.*T1'